test/unit/blackduck-security-action/bridge-cli.test.ts

```diff
@@ -327,14 +327,14 @@ test('Latest URL Version failure', async () => {
   jest.spyOn(HttpClient.prototype, 'get').mockResolvedValueOnce(httpResponse)
 
   const sb = new Bridge()
-  const response = await sb.getBridgeVersionFromLatestURL('https://artifact.com/latest/bridge-cli.zip')
+  const response = await sb.getBridgeVersionFromLatestURL('https://artifact.com/latest/bridge-cli-bundle.zip')
   expect(response).toContain('')
 })
 
 test('Test fetch version details from BRIDGE_CLI_DOWNLOAD_URL', () => {
   const sb = new Bridge()
   Object.defineProperty(inputs, 'BRIDGE_CLI_DOWNLOAD_VERSION', {value: ''})
-  Object.defineProperty(inputs, 'BRIDGE_CLI_DOWNLOAD_URL', {value: 'https://test-url/bridge-cli-0.1.1-macosx.zip'})
+  Object.defineProperty(inputs, 'BRIDGE_CLI_DOWNLOAD_URL', {value: 'https://test-url/bridge-cli-bundle-0.1.1-macosx.zip'})
```

Comment:
can we add some test cases for linux and windows as well?

Reply:
made the changes please check